tests/core_tests/test_path_cache.py

```diff
@@ -733,7 +746,7 @@ def test_concurrent_full_sync(self):
         while not all_processes_finished:
             time.sleep(0.1)
             sys.stderr.write(".")
-            all_processes_finished = all([not(p.is_alive()) for p in processes])
+            all_processes_finished = all(not(p.is_alive()) for p in processes)
```

Comment:
A generator can be used directly inside a single argument function call. Thought I'd propagate the pattern here.